.github/workflows/rust.yml

```diff
@@ -24,22 +24,27 @@ jobs:
     runs-on: ubuntu-latest-16core
     container: ghcr.io/famedly/rust-container:nightly
     steps:
-    - uses: actions/checkout@v3
-    
-    - uses: ./.github/actions/rust-prepare
-      with:
-        gitlab_ssh: ${{ secrets.CI_SSH_PRIVATE_KEY}}
-        gitlab_user: ${{ secrets.GITLAB_USER }}
-        gitlab_pass: ${{ secrets.GITLAB_PASS }}
-        additional_packages: ${{ env.ADDITIONAL_PACKAGES }}
+      - uses: actions/checkout@v3
+      
+      - uses: ./.github/actions/rust-prepare
+        with:
+          gitlab_ssh: ${{ secrets.CI_SSH_PRIVATE_KEY}}
+          gitlab_user: ${{ secrets.GITLAB_USER }}
+          gitlab_pass: ${{ secrets.GITLAB_PASS }}
```

Comment:
If we update all the git dependencies across our repos, we could probably get rid of the gitlab keys and such here, right?

Reply:
I think we should wait until that is done everywhere :D